src/features/global/TabNotifications.js

```diff
@@ -0,0 +1,62 @@
+const Feature = require("../../core/Feature");
+const Global = require("../../sections/Global");
+const ConfigurableArray = require("../../core/ConfigurableArray");
+const Text = require("../../configurables/Text");
+const Checkbox = require("../../configurables/Checkbox");
+const Util = require("../../core/Util");
+
+class TabNotifications extends Feature {
+  constructor() {

```

Comment:
Nice work! Give yourself credit by adding the author property.
You can find an example in the contributing.md or other features.